src/lib/db/clients/mysql.js

```diff
@@ -178,6 +180,13 @@ export async function getTableReferences(conn, table) {
   return data.map((row) => row.referenced_table_name);
 }
 
+export async function getPrimaryKey(conn, database, table) {
+  logger().debug('finding foreign key for', database, table)
+  const sql = `SHOW KEYS FROM ${table} WHERE Key_name = 'PRIMARY'`
+  const { data } = await driverExecuteQuery(conn, { query: sql })
+  return data[0] ? data[0].Column_name : null
```

Comment:
@MasterOdin Is this one of those things that will be capitalized in mysql8?

Reply:
Yes its starts with a capitalized `C`